### PR TITLE
WRQ-17725: Updates node modules' major versions

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updates node modules' major versions

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updates node modules' major versions

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Migrated lerna.json with the added "$schema"
Added node engine v18 in root package.json because it is required by lerna
Updated node engine version for cordova to v14.14 because it is required by fs-extra
Updated electron version in browserslist

### Links
[//]: # (Related issues, references)
WRQ-17725

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,6 @@
   ],
   "devDependencies": {
     "eslint-config-enact-proxy": "^1.0.7",
-    "lerna": "^7.4.2"
+    "lerna": "^8.1.2"
   }
-}
\ No newline at end of file
+}
```
Comment (daniel-stoian-lgp) [inline package.json:34]: please add a block for node restrictions, vased on the breaking changes

image

File: package.json [hunk 2]
```diff
@@ -9,6 +9,9 @@
     "url": "https://github.com/enactjs/templates.git"
   },
   "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14.14.0"
```
Comment (daniel-stoian-lgp) [inline package.json]: please put the node version supported by lerna

**Review by daniel-stoian-lgp — APPROVED** [review]

File: lerna.json [hunk 3]
```diff
@@ -12,5 +11,6 @@
         "--no-package-lock"
       ]
     }
-  }
+  },
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json"
```
Comment (hong6316) [inline lerna.json:15]: Please note this change in `Additional Considerations`

File: package.json [hunk 4]
```diff
@@ -9,6 +9,9 @@
     "url": "https://github.com/enactjs/templates.git"
   },
   "license": "Apache-2.0",
+  "engines": {
+    "node": ">=18.0.0"
```
Comment (hong6316) [inline package.json:13]: Please note this change in `Additional Considerations`

File: packages/cordova/package.json [hunk 5]
```diff
@@ -12,14 +12,14 @@
   "homepage": "https://github.com/enactjs/templates/tree/master/packages/cordova#readme",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=6.4.0"
+    "node": ">=14.14.0"
```
Comment (hong6316) [inline packages/cordova/package.json:15]: Please note this change in `Additional Considerations`

File: packages/electron/template/package.json [hunk 6]
```diff
@@ -52,7 +52,7 @@
     "@enact/spotlight": "^4.8.0",
     "@enact/ui": "^4.8.0",
     "@enact/webos": "^4.8.0",
-    "electron": "^28.2.10",
+    "electron": "^30.0.1",
```
Comment (hong6316) [inline packages/electron/template/package.json:55]: Please check line 32~34(browserslist)
Especially, I found `Electron 27.x.y has reached end-of-support` in https://github.com/electron/electron/releases/tag/v30.0.0 . Is it OK?

Also, please note this change in `Additional Considerations` when you change it.

Comment (paul-beldean-lgp) [reply packages/electron/template/package.json:55]: I updated it to v28 to be safe

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**